app/scripts/background.js

```diff
@@ -2,7 +2,14 @@
  * @file The entry point for the web extension singleton process.
  */
 
-// this needs to run before anything else
+
+// these need to run before anything else
+if (
```

Comment:
You could move this condition into the `freezePromise` file for a slightly tighter entry file.

Reply:
Done